landoapi/workers/landing_worker.py

```diff
@@ -104,6 +104,7 @@ def loop(self):
 
         if job is None:
             self.throttle(self.sleep_seconds)
+            db.session.commit()
```

Comment:
Nit: should we commit and then throttle? Seems like we're just holding the lock for `sleep_seconds` longer than we need to?